develop/buildkite/pipeline.yml

```
@@ -11,12 +11,6 @@ steps:
 
   - wait
 
-  - label: ":docker: build docker images"
-    agents:
-      queue: "default"
-      docker: "*"
-    command: "docker build ."
-
```

Yes, we can't build docker here any more but you also shouldn't remove it. Let's change this to:
```suggestion
  - label: ":docker: build without cgo"
    agents:
      queue: "default"
      docker: "*"
    command: "CGO_ENABLED=0 make bins"
    plugins:
      - docker-compose#v3.8.0:
          run: build
          config: ./develop/buildkite/docker-compose.yml    
```